### PR TITLE
Update project setup and port to Python 3

### PR DESCRIPTION
**Pull request by jodal**
This patch series follows the checklist I've used on the extensions in the Mopidy organization, without any special treatment: https://gist.github.com/jodal/6c011e327035cc4ad56dbc2e3c28f1eb

`tox` with all envs runs clean on my system. You must enable CircleCI, like described in the checklist, before this PR can get a run there.

### CODE REVIEW

**Comment by jodal** [comment]
@tkem Anything you don't like here or would like done differently before I give the same treatment to your other extensions?

File: README.rst [hunk 1]
```diff
@@ -12,46 +29,23 @@ network, so there is no configuration needed.
...
-License
-------------------------------------------------------------------------
-
-Copyright (c) 2015-2018 Thomas Kemmer.
 
-Licensed under the `Apache License, Version 2.0`_.
+Project resources
+=================
 
+- `Source code <https://github.com/tkem/mopidy-dleyna}>`_
```
Comment (tkem) [inline README.rst]: Closing curly brace looks odd.

File: README.rst [hunk 2]
```diff
@@ -12,46 +29,23 @@ network, so there is no configuration needed.
...
+- `Issue tracker <https://github.com/tkem/mopidy-dleyna}/issues>`_
```
Comment (tkem) [inline README.rst]: Closing curly brace looks odd.

File: README.rst [hunk 3]
```diff
@@ -12,46 +29,23 @@ network, so there is no configuration needed.
...
+- `Changelog <https://github.com/tkem/mopidy-dleyna}/blob/master/CHANGELOG.rst>`_
```
Comment (tkem) [inline README.rst]: See above.

**Comment by tkem** [comment]
@jodal: Apart from some (apparently) stray curly braces in README links, nothing to object on first (rather quick) review. I always preferred single quotes for strings in Python code, but if that's `black`'s default, so be it.

File: README.rst (hunk 1 above)
Comment (jodal) [reply README.rst]: Good catch!

**Comment by jodal** [comment]
Yeah, I totally see where you're coming from. I also used to strongly prefer single quotes, and my initial reaction to Black was negative because of this single thing.

In a couple of major work projects we've first introduced Black without string normalization and months later turned on string normalization too, going all in with "Black, no exceptions". We've found that leaving all control to autoformatters, Black in Python and Prettier on the frontend, has basically only benefits. You get very quickly used to whatever the formatter decides on. Consistency is more important than our personal nitpicks.

I still use single quotes when I code, as they're a keypress less on US layouts, but my editors all format on save and fixes the quotes without me noticing.

**Comment by tkem** [comment]
@jodal: I tried to enable CircleCI but get the following error:
```
Orb codecov/codecov@1.0.5 not loaded. To use this orb, an organization admin must opt-in to using third party orbs in Organization Security settings.
```
Any ideas?

**Comment by jodal** [comment]
1. https://circleci.com/gh/organizations/tkem/settings#security
2. Check "Yes, allow all members of my organization to publish dev orbs, use uncertified orbs, and use third-party orbs (not supported by CircleCI) in project configuration."

Orbs is a CircleCI thing to reduce repetition in CircleCI config files, by reusing existing orbs for some steps of the workflows.

Let me know when you've changed it, and I can force-push again to create a new fresh build.

**Comment by tkem** [comment]
I already checked "Yes, allow all members of my organization to publish dev orbs, use uncertified orbs, and use third-party orbs (not supported by CircleCI) in project configuration." and tried to "rerun" the job, but no change - so do you have to force-push for this to take effect?

**Comment by jodal** [comment]
Yeah, as the CircleCI error message says: "Don't rerun this job. Rerunning will have no effect."

I just added these steps to the extension check list.

**Comment by jodal** [comment]
:tada:

**Comment by jodal** [comment]
BTW, if you "follow" the other repos on CircleCI now, you'll get an initial failed build because the master branch has no config, but my future PRs will have somewhere to build right away.

**Comment by tkem** [comment]
Already did that and checked the "Build forked pull requests" to "On" (I think).

**Comment by tkem** [comment]
Merged into [develop](https://github.com/tkem/mopidy-dleyna/tree/develop) branch, thank you!

**Pull request closed without merging**